build.sbt

```diff
@@ -21,7 +21,11 @@ lazy val root = (project in file("."))
     fileDescriptorLimit := Some("16384"),
     maintainer := "Guardian Developers <dig.dev.software@theguardian.com>",
     topLevelDirectory in Universal := Some(normalizedName.value),
+    riffRaffPackageName := s"devx::${name.value}",
+    riffRaffManifestProjectName := riffRaffPackageName.value,
     riffRaffPackageType := (packageBin in Debian).value,
+    riffRaffUploadArtifactBucket := Option("riffraff-artifact"),
+    riffRaffUploadManifestBucket := Option("riffraff-builds"),
```

Comment:
It would be nice if the plugin would default these buckets for the `riffRaffUpload` task (they don't seem to be needed for the `riffRaffNotifyTeamcity` task...)

Reply:
https://github.com/guardian/sbt-riffraff-artifact/pull/63